### PR TITLE
Create resolve-path-using-msvcrt.yml

### PR DESCRIPTION
**Pull request by re-fox**
Should address https://github.com/fireeye/capa-rules/issues/82

### CODE REVIEW

File: host-interaction/cli/resolve-path-using-msvcrt.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  meta:
+    name: resolve path using msvcrt
+    namespace: host-interaction/cli
+    author: "@_re_fox"
+    scope: basic block
+    examples:
+      - 31600ad0d1a7ea615690df111ae36c73:0x4016B8
+  features:
+    - or:
+      - api: msvcrt.__p__pgmptr
```
Comment (mr-tz) [inline host-interaction/cli/resolve-path-using-msvcrt.yml:11]: let's add wpgmptr, _get_pgmptr and _get_wpgmptr?

see https://docs.microsoft.com/en-us/cpp/c-runtime-library/pgmptr-wpgmptr?view=vs-2019

Comment (re-fox) [reply host-interaction/cli/resolve-path-using-msvcrt.yml:11]: Yep, thanks for the spot.  I found the following ones crawling the exports.
```
   d8    35e40  __p__pgmptr
   e1    35ed0  __p__wpgmptr
  1c4    66310  _get_pgmptr
  1ca    66440  _get_wpgmptr
  316    b6074  _pgmptr
  460    b6078  _wpgmptr
```
I'll add those in.

Comment (mr-tz) [reply host-interaction/cli/resolve-path-using-msvcrt.yml:11]: awesome, thanks a lot!

**Review by mr-tz — APPROVED** [review]

**Comment by williballenthin** [comment]
sidebar: @re-fox has been added as a contributor to the following repos:
  - capa: READ
  - capa-rules: WRITE
  - capa-testfiles: WRITE

I don't think our process should change much: we like to have at least two sets of eyes on every non-trivial change, so please continue to submit PRs as you've been doing. (things like typos, etc. *could* be committed directly). Once a PR receives an "approval" then we're usually good to merge it directly, though sometimes we keep it open for other reviewers' awareness.

Since GH will recognize @re-fox as a contributor, we can now tag them to review PRs, too. Given their contributions to the project, we trust their feedback on proposed additions and changes. Especially for capa-rules, lets request a review from @re-fox when appropriate. @re-fox please also feel encouraged to volunteer responses to newly opened issues and feature requests. Thanks for your hard work!

**Comment by mr-tz** [comment]
Awesome! Congrats @re-fox 🥇

**Pull request merged**